src/main/java/nl/topicus/mssql2monetdb/CopyTable.java

```diff
@@ -242,4 +248,73 @@ public String getToViewSql()
 
 		return sql;
 	}
+
+	public String getFromColumns() {
```

Comment:
Nee, in principe niet. Ik weet niet of de Topicus formatting OpenSource is / publiek beschikbaar? Dit project is namelijk open voor iedereen.